install.sh

```diff
@@ -0,0 +1,101 @@
+#/bin/bash
+# The script requires root permissions
+
+function install_trento() {
+    . /etc/os-release
+    if [[ ! $NAME =~ "SUSE" ]]; then
+        echo "Operating system is not supported."
+        return -1
+    fi
+    case "$VERSION_ID" in
+    "15.3")
+        REPO=${REPO:-"https://download.opensuse.org/repositories/devel:/sap:/trento/15.3/devel:sap:trento.repo"}
+        ;;
+    *)
+        echo "Operating system is not supported."
+        return -1;;
+    esac
+    path=${REPO%/*}/
+    if zypper lr --details | cut -d'|' -f9 | grep $path; then
+        echo "$path repository already exists. Skipping."
+    else
+        zypper ar $REPO
+    fi
+    zypper ref
+    if which trento; then
+        echo "trento is already installed. Skipping."
+    else
+        zypper in -y trento
+    fi
+}
+
+function install_consul() {
+    if which consul; then
+        echo "consul is already installed. Skipping."
+        return 0
+    fi
+    if ! which unzip; then
+        zypper in -y unzip
```

Comment:
what happens here if we're not running in SUSE?

Reply:
It will fail. Shouldn't it?

Reply:
I got what you mean. The SLES has NAME="SLES". Now the script checks the PRETTY_NAME. Either it's leap or sle, its always SUSE ... .

Reply:
That's not what I meant: the check on the distro name is only on the `install_trento` function, so the `install_consul` one is not handling this fail condition gracefully!